src/main/resources/templates/layouts/animal_details_layout.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html xmlns:layout="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org">
```

Comment:
Format this file please, so it is better readable

Reply:
I will. If you mean putting each name space in different line.